src/v/cloud_storage/tests/tx_range_manifest_test.cc

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
can you please also add a roundtrip json -> tx_range_manifest -> json? It really helps me that I can get example of valid json manifest in test